crates/typst-library/src/text/mod.rs

```diff
@@ -215,6 +215,17 @@ pub struct TextElem {
     #[default(Rel::one())]
     pub spacing: Rel<Length>,
 
+    /// Whether to automatically insert spacing between CJK and Latin characters.
+    ///
+    /// ```example
+    /// #set text(cjk-latin-spacing: auto)
+    /// 第4章介绍了基本的API。
+    ///
+    /// #set text(cjk-latin-spacing: none)
+    /// 第4章介绍了基本的API。
+    /// ```
+    pub cjk_latin_spacing: Smart<Option<()>>,
```

Comment:
This should probably either be `Smart<Option<Never>>` or `Smart<NoneValue>` or `Option<AutoValue`, but currently it has a third state although there should only be two.

Reply:
Oh I didn't realized the `Never` type. Fixed.